test/e2e/specs/onboarding/setup__build.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * @group calypso-pr
```

Comment:
**suggestion:nonblocking**: I think running this test on every commit for every branch is a bit overkill, but we can merge it as is and observe whether it causes problems.

Reply:
Yes, let's observe whether there is any potential issue 👍